js-core-2/lesson7.md

```diff
@@ -32,59 +32,60 @@ When we are you solving a problem, one of your main decisions as a developer is
 
 ## Object Exercises
 
-Checking if Property or Value Exists
+Exercise
+Write a function "printProperties" that takes an object and prints its properties
 
 ```js
-// Exercise
-// Write a function "printProperties" that takes an object and prints its properties
-var student = { 
-    name: "Simon", 
-    age: "21", 
+var student = {
+    name: "Simon",
+    age: "21",
     interests : ["javascript", "react"]
 };
 
 printProperties(student);
 // Output in this case should be name, age, interests
 ```
 
+Exercise
+Write a function called hasProperty that takes an object and a property
+the function should return true if the property exists, false if it doesn't
+
 ```js
-// Exercise
-// Write a function called hasProperty that takes an object and a property
-// the function should return true if the property exists, false if it doesn't
-var student = { 
-    name: "Simon", 
-    age: "21", 
+var student = {
+    name: "Simon",
+    age: "21",
     interests : ["javascript", "react"]
 };
 
 hasProperty(student, 'age'); // should return true
 hasProperty(student, 'job'); // should return false
 ```
 
-> Exercise: compare that to how you find an element in array?
+Exercise: compare that to how you find an element in array?
```

Comment:
Maybe try and format these exercises to stand out a bit from the rest of the theory, see below:

![image](https://user-images.githubusercontent.com/5975076/28118616-a49f33ca-6709-11e7-99c4-81cd64ef7b53.png)